docs/dev-env-first-time-contributors.md

```diff
@@ -584,7 +584,7 @@ Next, read the following to learn more about developing for Zulip:
 
 * [Git & GitHub Guide][rtd-git-guide]
 * [Using the Development Environment][rtd-using-dev-env]
-* [Testing][rtd-testing]
+* [Testing][rtd-testing] (and [Configuring Travis CI][travis-ci])
```

Comment:
I think this needs a bit more explanation, e.g. "Configuring Travis CI to test your fork so you don't need to create a pull request to get your code tested"